tests/create_policy/main.tf

```
@@ -20,10 +20,8 @@ resource "random_id" "name" {
 module "create_policy" {
   source = "../../"
 
-  acl    = "private"
+  create_policy = true
+
   bucket = random_id.name.hex
   policy = join("", data.template_file.this.*.rendered)
```

I realize this was here before, but this is some legacy syntax, and a legacy data source. Suggest:

```suggestion
  policy = templatefile("${path.module}/templates/config_bucket_policy.json", {
    bucket    = random_id.name.hex
    partition = local.partition
  })
```

And remove the `template_file` data source.

---

Updated to new syntax and removed the template_file data source.  This is a much cleaner implementation.